packages/ui-plugin-sheets/src/services/status-bar.service.ts

```diff
@@ -0,0 +1,47 @@
+import { FUNCTION_NAMES } from '@univerjs/base-formula-engine';
+import { Nullable } from '@univerjs/core';
+import { createIdentifier, IDisposable } from '@wendellhu/redi';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+export interface IStatusBarService {
+    state$: Observable<Nullable<IStatusBarServiceStatus>>;
+    dispose(): void;
+    setState(param: IStatusBarServiceStatus | null): void;
+    getState(): Readonly<Nullable<IStatusBarServiceStatus>>;
+    getFunctions(): Readonly<FUNCTION_NAMES[]>;
+}
+
+export type IStatusBarServiceStatus = Array<{
+    func: FUNCTION_NAMES;
+    value: number;
+}>;
+
+export class StatusBarService implements IStatusBarService, IDisposable {
+    private readonly functions = [
+        FUNCTION_NAMES.SUM,
+        FUNCTION_NAMES.MAX,
+        FUNCTION_NAMES.MIN,
+        FUNCTION_NAMES.AVERAGE,
+        FUNCTION_NAMES.COUNT,
+    ];
+    private readonly _state$ = new BehaviorSubject<Nullable<IStatusBarServiceStatus>>(null);
```

Comment:
IMHO the status should be stored in the local storage and get loaded from the local storage when the web page refreshes.

Reply:
The configuration will not be cached locally. Actually, status bar will display three or one statistical item by default based on the current size. Reference -> StatusBar.tsx 😄